### PR TITLE
Updated dev_cluster setup README to match latest updates

### PR DESCRIPTION
**Pull request by deepak1725**
Added PR Steps and General FAQ's

### CODE REVIEW

File: openshift/README-AWS_Access.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Open Merge Request for AWS Credentials
+Next step is to get credentials to signin in your dev AWS Console
```
Comment (samuzzal-choudhury) [inline openshift/README-AWS_Access.md]: **Correction**: Next step is to get credentials to get access to your own AWS console.

File: openshift/README-AWS_Access.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+1. Make sure your Github/Gitlab account has primary email as `xyz@redhat.com`
+2. Open a MR at https://gitlab.cee.redhat.com/dtsd/devguide/blob/master/devguide.md#requesting-access
```
Comment (samuzzal-choudhury) [inline openshift/README-AWS_Access.md]: `Create a Merge Request` instead of `Open a MR`.

File: openshift/README-AWS_Access.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+   Example MR (requesting dev_cluster and dev_staging access): https://gitlab.cee.redhat.com/service/app-interface/merge_requests/608/diffs#diff-content-e1e2398297f1033c8844202f9135ea408b397d08
+3. Repo would likely be forked.
```
Comment (samuzzal-choudhury) [inline openshift/README-AWS_Access.md]: Could not get what this step means

File: openshift/README-AWS_Access.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+## Open Merge Request for AWS Credentials
+Next step is to get credentials to signin in your dev AWS Console
+1. Make sure your Github/Gitlab account has primary email as `xyz@redhat.com`
+2. Open a MR at https://gitlab.cee.redhat.com/dtsd/devguide/blob/master/devguide.md#requesting-access
+
+   Example MR (requesting dev_cluster and dev_staging access): https://gitlab.cee.redhat.com/service/app-interface/merge_requests/608/diffs#diff-content-e1e2398297f1033c8844202f9135ea408b397d08
+3. Repo would likely be forked. 
+4. Before trying to put a Merge Request, make sure to add `@devtools-bot` with role as **Maintainer** in your forked repo settings.
```
Comment (samuzzal-choudhury) [inline openshift/README-AWS_Access.md]: Replace `trying to put` with `creating`

File: openshift/README-AWS_Access.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+## Open Merge Request for AWS Credentials
+Next step is to get credentials to signin in your dev AWS Console
+1. Make sure your Github/Gitlab account has primary email as `xyz@redhat.com`
+2. Open a MR at https://gitlab.cee.redhat.com/dtsd/devguide/blob/master/devguide.md#requesting-access
+
+   Example MR (requesting dev_cluster and dev_staging access): https://gitlab.cee.redhat.com/service/app-interface/merge_requests/608/diffs#diff-content-e1e2398297f1033c8844202f9135ea408b397d08
+3. Repo would likely be forked. 
+4. Before trying to put a Merge Request, make sure to add `@devtools-bot` with role as **Maintainer** in your forked repo settings.    
+5. With your new `kerbroseid.yml` file, Put a Merge Request in Base Repo while ensuring that all automated tests are passed.
```
Comment (samuzzal-choudhury) [inline openshift/README-AWS_Access.md]: Replace `Put` with `create`.

File: openshift/README-AWS_Access.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+6. Once Merged, You should receive email from **App SRE team automation**, specifying your dev_console's AWS credentials.
+7. In Addition you should also receive invitation email from `@app-sre-bot` to join `@rhdt-dev` and `@app-sre` organisation.
+(Based on permissions requested). Make sure to accept this invite. This will authorise you in dev_cluster signin.
+
+**Please Note**: Your `dev_cluster` resource usage can be seen in AWS console.  
+ 
+ Go to [README.md](README.md)
```
Comment (samuzzal-choudhury) [inline openshift/README-AWS_Access.md:20]: @deepak1725 please take this text through a grammar checker browser extension.

**Review by samuzzal-choudhury — CHANGES_REQUESTED** [review]

File: openshift/README-AWS_Access.md (hunk 3 above)
Comment (deepak1725) [reply openshift/README-AWS_Access.md]: reworded.

File: openshift/README-AWS_Access.md (hunk 6 above)
Comment (deepak1725) [reply openshift/README-AWS_Access.md:20]: Noted

File: openshift/README.md [hunk 7]
```diff
@@ -54,6 +85,14 @@ $ ./cleanup.sh
...
 to remove the OpenShift project and all allocated AWS resources.
 
+### FAQ's:
+1. In dev_console, `bayesian-data-importer` service is down.
+
+**cause**: In this case you have some data messed up in your Dynamo DB Tables.
+**resolution**: Completely remove **your tables only** from AWS Dynamo DB, i.e  tables prefixed with name `your_kerboros_*`. 
+Redeploy `bayesian-gremlin-http` to recreate tables and then redeploy bayesian-data-importer.
+
+
```
Comment (maheshwari-git) [inline openshift/README.md:96]: Removal of the below was required for execution,
1. RDS creation in helper.sh
2. --db-subnet-group-name "${RDS_SUBNET_GROUP_NAME}" \

Comment (maheshwari-git) [inline openshift/README.md:96]: Once the Pod's up need to value the services by triggering API endpoint 
example : Component Analysis
http://<"your-api-endpoint">/api/v1/component-analyses/npm/serve-static/1.7.1
example : Stack Analysis

Comment (maheshwari-git) [inline openshift/README.md:96]: For validating your stack analysis call, use Postman 

Headers which need to be set are
key : x-3scale-account-secret  value: 
key : Content-Type value: application/x-www-form-urlencoded
key : ecosystem value : <"npm">/<"pyp"i>
key: origin value:vscode

Body
key : manifest[]  value : "pom.xml" / "package.json"
key: filePath[]  value : "some path" -this is not used in the api

Comment (maheshwari-git) [reply openshift/README.md:96]: For Stack Analysis call ensure your hpf-insights job is running. If it is not then you need to launch the service using the "deploy using yaml file" option on your openshift console.

Pick the yaml file from your cloned repo f8a-hpf-insights/openshift/template.yaml

Comment (deepak1725) [reply openshift/README.md:96]: I think i should put a separate PR removing this..
https://github.com/fabric8-analytics/fabric8-analytics-deployment/blob/cc58a9abaaf18b99857f913e9cd42387a3e91ce4/openshift/helpers.sh#L104

**Review by samuzzal-choudhury — APPROVED** [review]
@deepak1725 Thank you for the PR. Please merge this.

**Pull request merged**